.github/workflows/periodics-daily.yaml

```diff
@@ -19,6 +19,7 @@ jobs:
         stale-issue-label: 'no-issue-activity'
         stale-pr-label: 'no-pr-activity'
         exempt-issue-labels: 'good-first-issue'
+        days-before-close: 21
```

Comment:
Can you also update L12 to match, thank you in advance!

Reply:
Thanks, good catch!